commons/src/main/java/com/expedia/www/haystack/pipes/commons/kafka/config/KafkaProducerConfig.java

```diff
@@ -0,0 +1,77 @@
+package com.expedia.www.haystack.pipes.commons.kafka.config;
+
+import org.apache.kafka.clients.CommonClientConfigs;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.common.serialization.StringSerializer;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class KafkaProducerConfig {
+
+    private String brokers;
+
+    private int port;
+
+    private String toTopic;
+
+    private String acks;
+
+    private int batchSize;
+
+    private int lingerMs;
+
+    private int bufferMemory;
+
+    public KafkaProducerConfig(final String brokers, final int port, final String toTopic, final String acks, final int batchSize, final int lingerMs, final int bufferMemory) {
+        this.brokers = brokers;
```

Comment:
and here ... well, you get my point :).

Reply:
same. Will be taking care of this in ProjectConfiguration